lsusb.c

```diff
@@ -117,12 +117,14 @@
 
 unsigned int verblevel = VERBLEVEL_DEFAULT;
 static int do_report_desc = 1;
-static const char * const encryption_type[] = {
-	"UNSECURE",
+static const char *const encryption_type[] = {
+	// clang-format off
+	"INSECURE",
 	"WIRED",
 	"CCM_1",
 	"RSA_1",
-	"RESERVED"
+	"RESERVED",
```

Comment:
This type of change is great, but it's not documented what you are doing this for here.  It's not a spelling change.